docs/TRACEBACKS.md

```diff
@@ -92,7 +92,9 @@ What the `print` calls revealed is that we used `/` when we should have used `//
 ## Logging
 
 [Logging][logging] can be used similarly to `print`, but it is more powerful.
-Logging can print to the console, but it can also be configured to print to a file.
+What is logged can be configured by the logging severity (e.g., 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL'.)
+A call to the `logging.error` function can pass `True` to the `exc_info` parameter, which will additionally log the stack trace.
+By configuring multiple handlers, logging can print to more than one place with the same logging function.
```

Comment:
Because of the closeness of the function with the word "print"` we might use "store or display" rather than "print".  "Output" might be another word, but that is often used even more "loosely".

Still, this is better.

Reply:
I see what you mean. Let me come up with a synonym...

Reply:
Maybe "persist"?

Reply:
That data persists, the logger does not.  So I do not think that is the right word.  The logger can write to standard out, standard error, or a file.  And "write" it does.

Reply:
Output? Write? Record? Logs aren't always printed (eg Datadog) nor persisted (mem based).